app/Models/Contact.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Contact extends Model
+{
+    // テーブル名
+    protected $table = 'contacts';
+
+    // 可変項目
+    protected $fillable =
+    [
+        'name',
+        'email',
+        'message',
+    ];
+}
```

Comment:
せっかくなので `$casts` も定義しましょうか。
意味はググってみてください。

```suggestion
    protected $casts = [
        'id' => 'integer',
        'name' => 'string',
        'email' => 'string',
        'message' => 'string',
    ];
}
```

Reply:
$castsでデータアクセス時に型変換してくれるんですね！
対応しました。